src/yadism/runner.py

```diff
@@ -146,7 +146,7 @@ def __init__(self, theory: dict, observables: dict):
             GF=theory["GF"],
             M2W=theory["MW"] ** 2,
             M2target=theory["MP"] ** 2,
-            fonll_parts=theory["fonll-parts"],
+            fonll_parts=theory["FONLLParts"],
```

Comment:
I believe you should call this `fonllparts` as in `coefficient_functions/__init__.py`